source/controlTypes/__init__.py

```diff
@@ -0,0 +1,236 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2007-2021 NV Access Limited, Babbage B.V.
+
+from addonAPIVersion import BACK_COMPAT_TO
+
+# imports used in __all__
+from .isCurrent import IsCurrent
+from .processing import OutputReason, processAndLabelStates, processNegativeStates, processPositiveStates
+from .role import Role, silentRolesOnFocus, silentValuesForRoles, _roleLabels
+from .state import State, STATES_SORTED, negativeStateLabels, _stateLabels
+
+
+__all__ = [
+	"IsCurrent",
+	"OutputReason",
+	"processAndLabelStates",
+	"processNegativeStates",
+	"processPositiveStates",
+	"Role",
+	"silentRolesOnFocus",
+	"silentValuesForRoles",
+	"State",
+	"STATES_SORTED",
+	"negativeStateLabels",
+]
+
+
+# Added to maintain backwards compatibility, marked for deprecation to be removed in 2022.1
+# usages to be replaced by Role.*.displayString and State.*.displayString
+if BACK_COMPAT_TO[0] < 2022:
+	roleLabels = _roleLabels
+	stateLabels = _stateLabels
+
+
+# Added to maintain backwards compatibility, marked for deprecation to be removed in 2022.1
+if BACK_COMPAT_TO[0] < 2022:

```

Comment:
I think realizing the deprecations, and removing access to these constants should happen as early as possible in the release dev cycle. This isn't updated until the end. NVDA version should be used instead. Though I am happy to hear arguments to the contrary.